src/pages/ProfileSettings.tsx

```diff
@@ -1,20 +1,261 @@
-import BillingCard from "@/templates/BillingCard";
-import ProfileCard from "@/templates/ProfileCard";
+import { Card } from "@/components/ui/card";
+import { useState } from "react";
+import { UserRound } from "lucide-react";
+import LabelWithText from "@/components/elements/LabelWithText";
+import { Button } from "@/components/ui/button";
+import { X } from "lucide-react";
+import InputWithLabel from "@/components/elements/InputWithLabel";
+import { Pencil } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import InputWithTagLeft from "@/components/elements/InputWithTagLeft";
 
 export default function ProfileSettings() {
+  const [image, setImage] = useState(null);
+
+  const onImageChange = (event: React.FormEvent<HTMLInputElement>) => {
+    if (event.currentTarget.files && event.currentTarget.files[0]) {
+      setImage(URL.createObjectURL(event.currentTarget.files[0]));
+    }
+  };
+
+  const clearImage = () => {
+    setImage(null);
+  };
+
   return (
     <>
-      <div className=" bg-gray-100">
-        <div className="font-semibold text-2xl text-center m-2 lg:text-left lg:ml-6">
-          <div className="bg-white">
-            <form>
-              {/* Profile Settings form */}
-              <ProfileCard />
-              {/* Billing Details */}
-              <BillingCard />
-            </form>
-          </div>
-        </div>
+      <div className="font-semibold m-6 text-left">
+        <Card className=" border-none shadow-none">
+          <form>
+            <Card className="mb-6 px-6 border-blue-50">
+              <div className="flex flex-col justify-center py-4 gap-y-4 text-left lg:grid lg:grid-cols-4 lg:items-center">
+                <div className="flex flex-row gap-4 items-center  justify-between lg:col-span-4 ">
+                  <div className="flex flex-row gap-x-4 items-center">
+                    {image ? (
+                      <img src={image} className="h-15 w-15 rounded-full p-1 z-0" />
+                    ) : (
+                      <svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <circle cx="30" cy="30" r="30" fill="#1F499E" />
+                        <path
+                          d="M20.6667 37.0002C20.6667 35.7625 21.1584 34.5755 22.0336 33.7003C22.9088 32.8252 24.0957 32.3335 25.3334 32.3335H34.6667C35.9044 32.3335 37.0914 32.8252 37.9666 33.7003C38.8417 34.5755 39.3334 35.7625 39.3334 37.0002C39.3334 37.619 39.0876 38.2125 38.65 38.6501C38.2124 39.0877 37.6189 39.3335 37.0001 39.3335H23.0001C22.3812 39.3335 21.7878 39.0877 21.3502 38.6501C20.9126 38.2125 20.6667 37.619 20.6667 37.0002Z"
```

Comment:
you can use lucide-react icons or create custom components for displaying svg, we can pass just fill color as prop to this custom svg component